inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/node/extract/KafkaExtractNodeTest.java

```diff
@@ -49,7 +51,12 @@ public KafkaExtractNode getTestObject() {
         List<FieldInfo> fields = Arrays.asList(
                 new FieldInfo("name", new StringFormatInfo()),
                 new FieldInfo("age", new IntFormatInfo()));
-        return new KafkaExtractNode("1", "kafka_input", fields, null, null, "workerCsv",
+        Map<String, String> properties = new HashMap<>();
+        List<String> auditIds = new ArrayList<>();
+        auditIds.add("7");
```

Comment:
Metric number, make it constant plz

Reply:
It looks like this magic number is still there, and it is recommended to extract it as a constant in this class, with an explanation of what it means.